tests/unittests/algo/test_evolution_es.py

```diff
@@ -504,10 +504,10 @@ def infer_repetition_and_rung(self, num: int):
 
         return 1, budgets.index(num)
 
-    def assert_callbacks(
-        self, spy, num: int, algo: SpaceTransformAlgoWrapper[EvolutionES]
-    ):
-
+    def assert_callbacks(self):
+        # TODO: Move this to wherever it belongs, probably in the cleanup of a fixture.
```

Comment:
It's not really a cleanup, it's more like a generic test that is added at the end of the tests. But it is not really necessary to run at end of all tests and could actually be a separate test on its own, using the phase parametrization too.

Reply:
I agree: What I was suggesting was to move this testing logic after the `yield` statement inside a fixture, in other words, in the "cleanup" phase, not that this is cleanup code.